app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,9 @@
+class RecipesController < ApplicationController
+  def index
```

Comment:
You wouldn't need an `index` action in your recipes controller. We never visit a `recipes` index page.